shared/css/sidebar.css

```diff
@@ -0,0 +1,93 @@
+.media-wrapper:not(.showing-local-streams) > .media-control-buttons {
```

Comment:
General note: the css feels like a mixture of reaching down from top levels into lower components, and some component items. I think we need to standardise on something - probably making layout within a component being defined from its css class, and layout of the component within other components being defined by the other components.

Happy for this to be a follow-up review.